jitbuilder/release/Makefile

```diff
@@ -137,6 +138,11 @@ simple : libjitbuilder.a Simple.o
 Simple.o: src/Simple.cpp src/Simple.hpp
 	g++ -o $@ $(CXXFLAGS) $<
 
+atom : libjitbuilder.a Atomic.o
```

Comment:
Do you imagine all atomic operations will be tested in this file? If so, perhaps a name like AtomicOperations.[ch]pp would be better than just "Atomic". If you only ever imagine atomic add being tested, then I think the name should reflect that.

I think I prefer AtomicOperations so we collect together meaningful primitive tests.